packages/orbit-components/src/Wizard/WizardStep/index.js

```diff
@@ -0,0 +1,246 @@
+// @flow
+import * as React from "react";
+import { css } from "styled-components";
+import { hex2rgb } from "@csstools/convert-colors";
```

Comment:
i suppose we can write that function, no need to add another dependency, which was updated last time 2 years ago :)

Reply:
I was wondering what to do as well, but reinventing the function seemed unnecessary; `@csstools/convert-colors` is tree-shakable, so I don't think there will be any bundle size repercussions, and as far as the time they were last updated, I assume it's because these calculations are pretty straightforward, and possibly not widely used, but I've been collaborating with Jonathan Neal a bit and have been following him for a long time, so I trust his code.

Reply:
We've got `convertHexToRgba` in `orbit-design-tokens`, so feel free to use it 🙂 
https://github.com/kiwicom/orbit-design-tokens/blob/master/src/convertHexToRgba.js